skywalking-sniffer/skywalking-toolkit-activation/skywalking-toolkit-opentracing-activation/src/main/java/com/a/eye/skywalking/toolkit/activation/opentracing/tracer/interceptor/ExtractCrossProcessByteBufferContextInterceptor.java

```diff
@@ -0,0 +1,36 @@
+package com.a.eye.skywalking.toolkit.activation.opentracing.tracer.interceptor;
+
+import com.a.eye.skywalking.api.Tracing;
+import com.a.eye.skywalking.model.ContextData;
+import com.a.eye.skywalking.plugin.interceptor.EnhancedClassInstanceContext;
+import com.a.eye.skywalking.plugin.interceptor.enhance.InstanceMethodInvokeContext;
+import com.a.eye.skywalking.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
+import com.a.eye.skywalking.plugin.interceptor.enhance.MethodInterceptResult;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+
+import javax.net.ssl.StandardConstants;
+
+/**
+ * Created by wusheng on 2017/1/3.
+ */
+public class ExtractCrossProcessByteBufferContextInterceptor implements InstanceMethodsAroundInterceptor {
```

Comment:
Use the same naming principle. It's an activation, or a interceptor.

The principle is here:
* Use interceptor, when it's a plugin.
* Use activation, when it actives an application toolkit.